playbooks/common/openshift-master/config.yml

```diff
@@ -88,7 +69,7 @@
         ha: "{{ openshift_master_ha | default(groups.oo_masters | length > 1) }}"
         master_count: "{{ openshift_master_count | default(groups.oo_masters | length) }}"
 
-- name: Determine if session secrets must be generated
+- name: Inspect state of first master session secrets and config
```

Comment:
I see what you're doing here now. The referenced error (*undefined variabled*) is because `oo_masters_to_config` is having the `l_clean_install` fact set whereas later on we're referencing `hostvars[groups.oo_first_master.0`.

Makes sense to me. :1st_place_medal: